StarTrek13/Ships/ship.dm

```diff
@@ -886,7 +888,37 @@
 	get_shieldgen()
 	if(!theship)
 		to_chat(user, "Your ship has been destroyed!")
-	var/mode = input("Tactical console.", "Do what?")in list("fly ship", "remove pilot", "shield control", "red alert siren")
+	var/mode = input("Tactical console.", "Do what?")in list("fly ship", "remove pilot", "shield control", "red alert siren", "starmap")
+
+	var/html = "\
+	<html>\
+		<body>\
+			<div>\
+				<table>\
+					<tr>"
+	for(var/obj/effect/landmark/warp_beacon/wb in world)
```

Comment:
you should probably make a list of these, **in world** loops are bad

Reply:
in warp_beacons , there be a global list for them

Reply:
well there you go 👯‍♂️